### ISSUE TITLE
pyqt4 and pyqt5 can no longer be installed simultaneously

### ISSUE THREAD

**Issue by sbourdeauducq** [NixOS/nixpkgs#74974]
```
$ cat bug.nix
let
  pkgs = import <nixpkgs> {};
in
pkgs.mkShell {
  buildInputs = [
    (pkgs.python3.withPackages(ps: [
      ps.pyqt4
      ps.pyqt5
    ]))
  ];
}
$ nix-shell bug.nix
these derivations will be built:
  /nix/store/8mwgh1w5v0ydd75wcb5ygf8mngx5qv53-python3-3.7.5-env.drv
building '/nix/store/8mwgh1w5v0ydd75wcb5ygf8mngx5qv53-python3-3.7.5-env.drv'...
collision between `/nix/store/xasr6qmp7hhchp45ysa20vj12813vfrm-python3.7-PyQt5.sip-4.19.18/lib/python3.7/site-packages/sipconfig.py' and `/nix/store/f2rr2k1hpf7br6kn3y1vwn7zc3033k39-python3.7-sip-4.19.18/lib/python3.7/site-packages/sipconfig.py'
builder for '/nix/store/8mwgh1w5v0ydd75wcb5ygf8mngx5qv53-python3-3.7.5-env.drv' failed with exit code 25
error: build of '/nix/store/8mwgh1w5v0ydd75wcb5ygf8mngx5qv53-python3-3.7.5-env.drv' failed
```

The problem is present with 19.09 (63c756db074d35a746244987f1ecf647c253d30d) but not present with 19.03 (5f7eae4bbb15ce7fa4da1984fd6ba5be7af50952).

**Comment by prusnak** [comment]
FTR this does not work even for the master

**Comment by sbourdeauducq** [comment]
still important to me

**Comment by hacxman** [comment]
This was so much blocker today that I had to use ubuntu ;-(

**Comment by doronbehar** [comment]
Using an updated sip might help. See: https://github.com/NixOS/nixpkgs/issues/99936

**Comment by sbourdeauducq** [comment]
Still an issue

**Comment by FliegendeWurst** [comment]
This was fixed at some point, it works on NixOS 21.11.
